test/test_samples.rb

```diff
@@ -90,6 +93,9 @@ def test_filename_listed
       end
 
       language.filenames.each do |filename|
+        # Kludge for an unusual edge-case; see https://bit.ly/41EyUkU
+        next if language.name == "Hosts File" && filename == "HOSTS"
```

Comment:
You'll need to consider the "INI" entry too. How about:

```suggestion
        next if ["Host File", "INI"].include?(language.name) && filename == "HOSTS"
```